plugins/org.jboss.reddeer.junit/src/org/jboss/reddeer/junit/requirement/Requirement.java

```diff
@@ -45,4 +47,19 @@
 	 * Clean up after requirement.
 	 */
 	void cleanUp();
+	
+	/**
+	 * Decides whether it is possible to accept specific requirement's configuration object or not
+	 * @return true, if declaration fits in
+	 */
+	default boolean isDeclarationAcceptable() { return true; }
+	
+	/* (non-Javadoc)
+	 * @see org.jboss.reddeer.junit.execution.IExecutionPriority#getPriority()
+	 */
+	@Override
```

Comment:
remove this comment on overridden methods, javadoc automatically uses javadoc from superclass...